final_prep/mod_zero_hero.rb

```diff
@@ -1,52 +1,68 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
-
+hero_name = "Oz Striker"
+special_ability = "Kenetic energy manipulation"
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
-
+greeting = "#{hero_name} here to save the day!"
+carchphrase = "#{special_ability} is the name of the game!"
 # Declare two variables - power AND energy - set to integers
-
+power = 1000
+energy = 731
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
-
+full_power = power * 500
+full_energy = energy + 150
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
+is_human = false
+identity_concealed = true
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
-
+arch_enemies = ["Pants Man", "Lazy Man", "Weirdo"]
+sidekicks = ["Iron Man", "Black Widow", "Captain Marvel"]
 # Print the first sidekick to your terminal
-
+puts sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+puts arch_enemies[2]
 # Write some code to add a new arch_enemy to the arch_enemies array
-
+arch_enemies.push("Dr. Evil")
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+puts arch_enemies
 # Remove the first sidekick from the sidekicks array
-
+sidekicks.shift()
```

Comment:
✅ Yes... does Ruby need the parenthesis? What happens if you do `sidekicks.shift()` vs `sidekicks.shift`. Try it out in your terminal using the command `irb` to enter ruby environment